.github/workflows/auto-generate.yml

```diff
@@ -0,0 +1,53 @@
+name: Auto Generate Types
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+  push:
+    branches: [master]
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      # setup
+      - uses: actions/checkout@v2
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        id: nvm
+      - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+      - name: Install dependencies
+        run: npm ci
+      - name: Get latest types
+        run: npm run get-latest-types
```

Comment:
Should this be invoked as a first step of `npm start -- --new-revisions-only`?

Reply:
Yes. Logic here is that first, we get latest types from git, then we run generator which checks if the new version is actually newer by revision.
Are you saying that we should invoke this inside of `cli.ts` when `--new-revisions-only` option is passed?

Reply:
> Are you saying that we should invoke this inside of `cli.ts` when `--new-revisions-only` option is passed?

Yes, that _is_ what I meant, although I left this comment before I saw how things were implemented in `app.ts`, so perhaps we leave this as-is for now?

Reply:
Yes, I think it should be fine, but it's a good idea for future refactoring 👍